addons/horizontal-mystuff-tabs/addon.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Horizontal tabs on My Stuff",
+  "description": "Moves the My Stuff navigation buttons (\"All Projects\", \"My Studios\", \"Trash\") to the top of the page, instead of keeping them on the left.",
+  "tags": ["community"],
```

Comment:
@mxmou Any reason this shouldn't be a theme?

Reply:
It could be - editor-stage-left has a similar purpose and is a theme.